### DISCUSSION TITLE
Step Return Codes / datastore / Gauge Interaction Question

### DISCUSSION THREAD

**Discussion by WilkieIT** [getgauge/taiko#1919] [General]
I have a situation where I need to perform several steps before I get a onetime URL. 

I would prefer not to have 1 huge step that performs all of these steps.

My project is a JS project.  

Is there a way to either store scenario values e.g. Guages' ScenarioDataStore or return values so they can be used by other steps?

If not I have a gauge .Net project that could do all the hard work if there is a way to call a taiko test passing in the URL?

**Comment by NivedhaSenthil** [comment]
If you are using GaugeJs you could use the dataStore as how it would be used in .Net project
